anthos-deployment/main.tf

```diff
@@ -0,0 +1,64 @@
+resource "google_container_cluster" "primary" {
+  provider = google-beta
+
+  name                     = var.primary_name
+  location                 = var.location
+  remove_default_node_pool = true
+  initial_node_count       = 1
+
+  release_channel {
+    channel = var.release_channel
+  }
+
+  depends_on = [google_project_service.container]
+}
+
+resource "google_container_node_pool" "primary_preemptible_nodes" {
```

Comment:
For a learning perspective, any reason why here we are not using the GKE TF module from `terraform-google-modules/kubernetes-engine`?

Reply:
I don't think our goal is to have production deployment more demo environment. We thought to use CFT initially though. The main issues is that id you want to addd something to CFT you need to make PR towards CFT and good luck with that :)

Reply:
My point was more around, for the other resources we are using a module but not for the GKE one, was just curious to know why ;)

Reply:
I think the next iteration is to use a module. I was just hacking a demo first. I will open a task.